src/tz390.java

```diff
@@ -7684,4 +7688,27 @@ public boolean check_java_version(){
 		}
 		return true;
 }
+public String getVersion() {
+
+    try (InputStream input = getClass().getClassLoader().getResourceAsStream("z390.properties")) {
+
+            Properties prop = new Properties();
+
+            if (input == null) {
+                System.out.println("Sorry, unable to find z390.properties");
+                return null;
+            }
+
+            //load a properties file from class path, inside static method
+            prop.load(input);
+
+            //get the property value and print it out
+            return prop.getProperty("version");
+
+        } catch (IOException ex) {
+            ex.printStackTrace();
+            return null;
+        }
+
+}
 }
```

Comment:
This is a good way to get the version value. Think about finding a way to build z390.properties without using "git desribe".